expression/constant.go

```diff
@@ -448,3 +449,13 @@ func (c *Constant) Vectorized() bool {
 	}
 	return true
 }
+
+// SupportReverseEval checks whether the builtinFunc support reverse evaluation.
+func (c *Constant) SupportReverseEval() bool {
+	return true
```

Comment:
Because when a `ScalarFunction` support reverse evaluation. For example, `t.a plus constant(1)`. It will judge `isChildrenSupportReverseEval`, then the constant should return true. But the function can not call `ReverseEval` of `Constant`. This should be guaranteed by the programmer.